scenarios/kubernetes_janitor.py

```diff
@@ -72,22 +72,19 @@ def clean_project(project, hours=24, dryrun=False):
     'kubernetes-scale', # Let it's up/down job handle the resources
 ]
 
-PR_PROJECTS = [
-    'k8s-jkns-pr-kubemark',
-    'k8s-jkns-pr-gce',
-    'k8s-jkns-pr-gci-gce',
-    'k8s-jkns-pr-gke',
-    'k8s-jkns-pr-gci-gke',
-    'k8s-jkns-pr-gci-kubemark',
-    'k8s-jkns-pr-bldr-e2e-gce-fdrtn',
-    'k8s-jkns-pr-gce-etcd3',
-    'k8s-jkns-pr-gci-bld-e2e-gce-fd',
-]
+PR_PROJECTS = {
+    'k8s-jkns-pr-kubemark':3,
+    'k8s-jkns-pr-gce':3,
+    'k8s-jkns-pr-gci-gce':3,
+    'k8s-jkns-pr-gci-kubemark':3,
+    'k8s-jkns-pr-bldr-e2e-gce-fdrtn':24,
```

Comment:
Just so the 'clean' portion of the janitor doesn't collide with the "down" portion of the jenkins build & deploy script, both the janitor and the jenkins job need to be on a different period. I'd prefer if this was something > 24, so 27 or 28.

Also, eventually disabling the janitor should be possible (with :0) since resource footprint is fragile under federation testing and we clean additional custom cleaning logic (i.e. DNS). Please coordinate with @madhusudancs.